src/main/kotlin/racingCar/Race.kt

```diff
@@ -0,0 +1,17 @@
+package racingCar
+
+class Race(private val carCount: Int, private val tryCount: Int) {
+    val carPositions = mutableListOf<Int>()
+    private val cars = mutableListOf<ICar>()
```

Comment:
```suggestion
    private val cars:List<Car> = (1..tryCount).map { Car(Random()) }
```
불변을 사용하지 않도록 해보는건 어떨까요?